src/filesystem.rs

```diff
@@ -0,0 +1,62 @@
+use std::process::Command;
+
+#[derive(Default, Debug, PartialEq)]
+pub struct FileSystem {
+    pub name: String,
+    pub used: u64,
+    pub size: u64,
+}
+
+pub fn get() -> std::io::Result<Vec<FileSystem>> {
+    let output = Command::new("df")
+        .arg("-Pkl")
+        .output()
+        .expect("failed to run");
+    collect_filesystem_stats(String::from_utf8(output.stdout).unwrap())
+}
+
+fn collect_filesystem_stats(output: String) -> std::io::Result<Vec<FileSystem>> {
```

Comment:
Use `R: std::io::Read` instead of `String` for adjusting to other module's `get` signature if you can. Is it difficult?